src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3332,4 +3332,58 @@ public async Task GetEnterpriseServerVersion_Returns_Version()
             .Replace("\\n", "")
             .Replace("\\t", "")
             .Replace(" ", "");
+
+    [Fact]
+    public async Task AbortMigration_Returns_Boolean()
+    {
+        // Arrange
+        const string url = "https://api.github.com/graphql";
+        const string migrationSourceId = "MIGRATION_SOURCE_ID";
+
+        const string query = @"
+                mutation abortRepositoryMigration(
+                    $migrationId: ID!,
+                )";
+        const string gql = @"
+                abortRepositoryMigration(
+                    input: { 
+                        migrationId: $migrationId
+                    }
+                ) {
+                    sucesss 
+                    }
+                  }";
```

Comment:
the reason why the test is failing is the payloads need to exactly match, there is a typo in success, and extra space and and extra `}` here. After you fixed those, you can use the `Compact` private method to compare the payload in the client mock setup.

Reply:
Cool I think i got them all!! Specs are passing now